sql/updates/world/2015_06_08_10_world.sql

```diff
@@ -0,0 +1,27 @@
+-- The column `spell1` was misused for this NPC. This NPC is undead. He cannot be MCed. The spell 29307 should be cast using SmartAI/ by Script.
+-- Also, link the zombies chows to the custom ScriptedAI defined in boss_gluth.cpp.
+UPDATE `creature_template` SET `spell1`=0, `ScriptName`='npc_zombie_chow' WHERE `entry`=16360;
+UPDATE `creature_template` SET `spell1`=0, `ScriptName`='npc_zombie_chow' WHERE `entry`=30303;
+
+-- connect the decimate spell (effect 0 & 2) used by Gluth (in both 10 man & 25) to the spell script defined in boss_gluth.cpp.
+-- and connect the 2 zombie chow search spells to a unique spell scripted use by both.
+DELETE FROM `spell_script_names` WHERE `spell_id` IN (28374, 54426);
```

Comment:
True. However, now that I think about it, the opposite is also true: A script can be called by more than one spell (I used one of those here). Therefore, I suppose I should specify both `ScriptName` & `spell_id`?